### PR TITLE
fix: Exporter alerts for DB connections

### PR DESCRIPTION
**Pull request by jacsamell**
# Description

-   Exporter alerts for DB connections

# Type of change

-   [ ] feat - A new feature
-   [X] fix - A bug fix
-   [ ] docs - Documentation only changes
-   [ ] codestyle - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] refactor - A code change that neither fixes a bug nor adds a feature
-   [ ] perf - A code change that improves performance
-   [ ] test - Adding missing tests or correcting existing tests
-   [ ] build - Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
-   [ ] ci - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] chore - Other changes that don't modify src or test files
-   [ ] revert - Reverts a previous commit
-   [ ] content - Changes to content or copy

### CODE REVIEW

File: fdbt-aws/cloudformation/service/eu-west-2/rds.yml [hunk 1]
```diff
@@ -70,6 +61,7 @@ Resources:
       Parameters:
         aurora_load_from_s3_role:
           Fn::ImportValue: !Sub ${Stage}:RdsAuroraS3RoleArn
+        max_connections: 90
```
Comment (JwanKhalaf) [inline fdbt-aws/cloudformation/service/eu-west-2/rds.yml:64]: How did we come up with `90`? Are we just guessing for the time being?

File: repos/exporter/lib/exporter-stack.ts [hunk 2]
```diff
@@ -64,7 +64,7 @@ export class ExporterStack extends cdk.Stack {
             logRetention: 180,
         });
 
-        this.addAlarmsToLambda(stage, exporterFunction, `exporter-${stage}`, 30000);
+        this.addAlarmsToLambda(stage, exporterFunction, `exporter-${stage}`, 300000);
```
Comment (JwanKhalaf) [inline repos/exporter/lib/exporter-stack.ts:67]: Can you explain this change, please? What does the `300000` mean or do?

File: repos/fdbt-netex-output/src/data/auroradb.ts [hunk 3]
```diff
@@ -23,7 +23,7 @@ export const getAuroraDBClient = (): Pool => {
             password: awsParamStore.getParameterSync('fdbt-rds-netex-output-password', { region: 'eu-west-2' }).Value,
             database: 'fdbt',
             waitForConnections: true,
-            connectionLimit: 10,
+            connectionLimit: 1,
```
Comment (JwanKhalaf) [inline repos/fdbt-netex-output/src/data/auroradb.ts:26]: Is this saying on prod, the connection limit is `1`?

Aren't databases able to handle 1000s of connections at the same time? I'm confused, can you shed some light on this, please?

File: repos/exporter/lib/exporter-stack.ts (hunk 2 above)
Comment (jacsamell) [reply repos/exporter/lib/exporter-stack.ts:67]: This takes the duration alarm from 30 seconds, to 5 minutes

File: fdbt-aws/cloudformation/service/eu-west-2/rds.yml (hunk 1 above)
Comment (jacsamell) [reply fdbt-aws/cloudformation/service/eu-west-2/rds.yml:64]: 90 is what the production database has by default, so bringing Test env in line with prod

File: repos/fdbt-netex-output/src/data/auroradb.ts (hunk 3 above)
Comment (jacsamell) [reply repos/fdbt-netex-output/src/data/auroradb.ts:26]: This means each Lambda will use 1 connection but we can have up to 40 instances running at the same time which will mean the connections are shared out between all the lambdas. The DB size we use only supports 90 connections

**Review by JwanKhalaf — APPROVED** [review]
:ok_hand: lovely explanations to my queries, thanks :pray:

**Pull request merged**